chips/ibex-super-system/openocd.cfg

```diff
@@ -0,0 +1,37 @@
+source [find mem_helper.tcl]
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
```

Comment:
This is differently licensed than rest of the hubris sources.

Reply:
Yea, It is a file provided by ibex_super_system project.

Reply:
I see..